<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GalPatternWidget</class>
 <widget class="QWidget" name="GalPatternWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>316</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Patterns</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="Pattern_comboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select known Sequence by name</string>
       </property>
       <item>
        <property name="text">
         <string>Alternating100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TwoSteps</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RandomNoise</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="PatternNewButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create new Sequence&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../galapicons.qrc">
         <normaloff>:/icons/filenew.png</normaloff>:/icons/filenew.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PatternEditButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Edit command code of selected sequence&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../galapicons.qrc">
         <normaloff>:/icons/analysiswin.png</normaloff>:/icons/analysiswin.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PatternLoadButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Load sequence code from file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../galapicons.qrc">
         <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PatternSaveButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Export current sequence code to file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../galapicons.qrc">
         <normaloff>:/icons/filesave.png</normaloff>:/icons/filesave.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PatternEditCancelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cancel edited code for selected sequence (restore last setup)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../galapicons.qrc">
         <normaloff>:/icons/undo.png</normaloff>:/icons/undo.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PatternApplyButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Apply command code for selected sequence&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../galapicons.qrc">
         <normaloff>:/icons/OK16.png</normaloff>:/icons/OK16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Okteta::ByteArrayColumnView" name="oktetabyteview">
     <property name="NoOfBytesPerLine">
      <number>8</number>
     </property>
     <property name="Coding" stdset="0">
      <enum>Okteta::AbstractByteArrayView::BinaryCoding</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Okteta::ByteArrayColumnView</class>
   <extends>QFrame</extends>
   <header>okteta/bytearraycolumnview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../galapicons.qrc"/>
 </resources>
 <connections/>
</ui>
